apps/frontend/src/app/5_pages/LeaderboardPointsPage/hooks/useGetPoints.ts

```diff
@@ -0,0 +1,60 @@
+import { useMemo } from 'react';
+
+import { useAccount } from '../../../../hooks/useAccount';
+import { areAddressesEqual } from '../../../../utils/helpers';
+import {
+  EXTRA_SPICE_POINTS_MULTIPLIER,
+  RUNES_POINTS_MULTIPLIER,
+} from '../LeaderboardPointsPage.constants';
+import { User, UserPoints } from '../LeaderboardPointsPage.types';
+import usersPointsList from '../data/usersPoints.json';
+
+export const useGetPoints = (pageSize: number, page: number) => {
+  const { account } = useAccount();
+  const usersPointsData: UserPoints[] = usersPointsList;
```

Comment:
```suggestion
  const data: UserPoints[] = usersPointsList;
```
Move it above the hook.